tests/scripts/test_zcbor.py

```diff
@@ -38,6 +37,7 @@
 p_manifest20 = Path(p_tests, 'cases', 'manifest20.cddl')
 p_test_vectors12 = tuple(Path(p_tests, 'cases', f'manifest12_example{i}.cborhex') for i in range(6))
 p_test_vectors14 = tuple(Path(p_tests, 'cases', f'manifest14_example{i}.cborhex') for i in range(6))
+p_test_vectors16 = tuple(Path(p_tests, 'cases', f'manifest14_example{i}.cborhex') for i in range(6))  # Identical to manifest14.
```

Comment:
If we use the same input, what is the purpose of the test?

Reply:
The examples are the same, but they are tested against the manifest14.cddl and manifest16.cddl files respectively.